app/models/automation_rule.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: automation_rules
+#
+#  id          :bigint           not null, primary key
+#  actions     :jsonb
+#  conditions  :jsonb
+#  description :text
+#  event_name  :string
+#  name        :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  account_id  :bigint           not null
+#
+# Indexes
+#
+#  index_automation_rules_on_account_id  (account_id)
+#
+class AutomationRule < ApplicationRecord
+  belongs_to :account
```

Comment:
we could probably add validations for supported event name, actions and conditions